README.md

```diff
@@ -79,6 +79,19 @@ const TestComponent = () => {
 }
 ```
 
+## Cookies
+
+**NOTE**: This section only applies to intent client versions `1.4.0` and up. If you install the intent client with the website tag, you automatically get access to the latest client version. Versions older than this use obfuscated cookie names. If for some reason you need access to these then you can reach out to the Unify team for support.
```

Comment:
You can get a nice [callout box](https://github.com/orgs/community/discussions/16925) with:

```suggestion
> [!NOTE]
> This section only applies to intent client versions `1.4.0` and up. If you install the intent client with the website tag, you automatically get access to the latest client version. Versions older than this use obfuscated cookie names. If for some reason you need access to these then you can reach out to the Unify team for support.
```